### ISSUE TITLE
*dbind-WARNING* *Couldn't connect to accessibility bus: Failed to connect to socket /root/.cache/at-spi/bus_1: Permission denied*

### ISSUE THREAD

**Issue by Benjamin-Loison** [Benjamin-Loison/pyzo#79]
<details>
<summary>Shell:</summary>

```
Python 3.11.2 (main, Nov 30 2024, 21:22:50) on linux (64 bits).

(python3:435678): dbind-WARNING **: 00:36:42.937: Couldn't connect to accessibility bus: Failed to connect to socket /root/.cache/at-spi/bus_1: Permission denied
This is the Pyzo interpreter with integrated event loop for PYSIDE6.
Type 'help' for help, type '?' for a list of *magic* commands.
```
</details>

on my Debian 12 GNOME laptop.

**Comment by Benjamin-Loison** [comment]
When *Terminate and restart the interpreter*.

**Comment by Benjamin-Loison** [comment]
```bash
ls -lh /root/.cache/at-spi/bus_1
```
```
ls: cannot access '/root/.cache/at-spi/bus_1': Permission denied
```

```bash
sudo ls -lh /root/.cache/at-spi/bus_1
```
```
srwxrwxrwx 1 root root 0 Apr 18 20:23 /root/.cache/at-spi/bus_1
```

I do not remember having proceeded to any related change recently.

https://pypi.org/project/pyzo/#history last update was on 17/01/25, as I update regularly pip packages, related to [Benjamin-Loison/pip/issues/13](https://github.com/Benjamin-Loison/pip/issues/13), the issue does not seem to come from Pyzo itself.

**Comment by Benjamin-Loison** [comment]
DuckDuckGo and Google search `"Couldn't connect to accessibility bus: Failed to connect to socket /root/.cache/at-spi/bus_1: Permission denied"`.

**Comment by Benjamin-Loison** [comment]
```bash
chmod o+rx /root/.cache/at-spi/
```
```
chmod: cannot access '/root/.cache/at-spi': Permission denied
```

<details>
<summary>Bash script:</summary>

```bash
sudo chmod o+rx /root/.cache/at-spi/
sudo chmod o+rx /root/.cache§
sudo chmod o+rx /root/
```
</details>

does not return anything.

Source: https://www.reddit.com/r/Fedora/comments/15rr3h9/evolution_does_not_start_after_upgrade_to_fedora/
Source: https://forums.fedoraforum.org/showthread.php?332478-Evolution-breaks-on-upgrade-to-F40-at-spi&p=1882433#post1882433

<details>
<summary>leads to (intermediary state do not):</summary>

```
Kernel process terminated for restart. (0)

Python 3.11.2 (main, Nov 30 2024, 21:22:50) on linux (64 bits).

(python3:496904): dbind-WARNING **: 20:43:14.618: Couldn't register with accessibility bus: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.
This is the Pyzo interpreter with integrated event loop for PYSIDE6.
Type 'help' for help, type '?' for a list of *magic* commands.

>>>
```
</details>

[darktable-org/darktable/issues/13521](https://github.com/darktable-org/darktable/issues/13521) does not seem helpful despite explicitly mentioning my error, see [darktable-org/darktable/issues/13521#issuecomment-1416545679](https://github.com/darktable-org/darktable/issues/13521#issuecomment-1416545679).

**Comment by Benjamin-Loison** [comment]
Should give a try to https://discuss.pixls.us/t/darktable-will-no-longer-start-from-gnome-icon-solved/32040/2.

**Comment by Benjamin-Loison** [comment]
```bash
ls -lha /root/.cache/at-spi/
```
<details>
<summary>Output:</summary>

```
total 8.0K
drwx---r-x  2 root root 4.0K Apr 18 20:23 .
drwx---r-x 15 root root 4.0K Apr 18 20:23 ..
srwxrwxrwx  1 root root    0 Apr 18 20:23 bus_1
```
</details>

**Comment by Benjamin-Loison** [comment]
```bash
sudo chmod o-rx /root/.cache/at-spi/
ls -lha /root/.cache/at-spi/
```
```
ls: cannot open directory '/root/.cache/at-spi/': Permission denied
```

Should try at this step `ls -lha /root/`, to verify that we just give anyone the ability to read `/root/` which can indeed be dangerous as mentioned by the Reddit initial post.

```bash
sudo chmod o-rx /root/.cache/
sudo chmod o-rx /root/
```

**Comment by Benjamin-Loison** [comment]
Unable to reproduce the issue after reboot.
